app/assets/javascripts/fae/vendor/trumbowyg/langs/ar.min.js

```diff
@@ -0,0 +1,11 @@
+/* ===========================================================
```

Comment:
While we have the ability to manage content in multiple languages in Fae, the interface only supports english currently. I'd prefer to keep all the lang files out and maybe supply a info in the docs to users can add them to the app if they so wish.